lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1794,6 +1843,177 @@ def cmd_sessions(*args)
     true
   end
 
+  def get_matching_sessions(search_term)
```

Comment:
@zgoldman-r7 I think @dwelch-r7 has called it out a few times now separately - but could we run Rubocop on all new code that's being added? i.e. these chunks of code, and the code in your test. You can run it from the command line with `bundle exec rubocop -A file_1.rb file_2.rb`

The rest of the file hasn't been rubocop'ed, so you'll just need to make sure the _new_ code is rubocop'ed